### PR TITLE
Fix CS

### PR DESCRIPTION
**Pull request by szepeviktor**
Obey coding standards.

@amjadr360

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,5 +22,10 @@
 		"lint": "@php ./vendor/bin/parallel-lint --exclude .git --exclude vendor .",
 		"standards:check": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
 		"standards:fix": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
+	},
+	"config": {
+		"allow-plugins": {
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		}
```
Comment (szepeviktor) [inline composer.json:29]: Composer v2 needs this.

File: phpstan.neon.dist [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
 includes:
-	- vendor/phpstan/phpstan/conf/bleedingEdge.neon
 	- vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
 	level: max
 	paths:
 		- functions.php
-		- inc/
+		- patterns/
```
Comment (szepeviktor) [inline phpstan.neon.dist:9]: `inc` was moved.

File: phpstan.neon.dist [hunk 3]
```diff
@@ -1,10 +1,9 @@
 #$ vendor/bin/phpstan analyze
 
 includes:
-	- vendor/phpstan/phpstan/conf/bleedingEdge.neon
```
Comment (szepeviktor) [inline phpstan.neon.dist:4]: This is dangerous 💥

**Comment by amjadr360** [comment]
@szepeviktor, thanks for the PR. The PR is removing the newline, but WPCS recommends having a newline at the end of the file. Could you please add the newline back?

**Comment by szepeviktor** [comment]
@amjadr360 There is a misunderstanding.

image

This means the new line is added.

**Comment by amjadr360** [comment]
I see, thank you for the clarification, Szepeviktor.

**Comment by szepeviktor** [comment]
Glad to contribute to your theme.

**Pull request merged**